src/stampy_nlp/utilities/huggingface_utils.py

```diff
@@ -6,26 +6,39 @@
 import logging
 import requests
 import os
+from dotenv import load_dotenv
+
+
+load_dotenv()
 
-HUGGINGFACE_API_KEY: str = os.getenv('HUGGINGFACE_API_KEY')
-if (HUGGINGFACE_API_KEY == None):
-    raise Exception("Missing environment variable HUGGINGFACE_API_KEY")
 
 HUGGINGFACE_API = 'https://api-inference.huggingface.co/pipeline/feature-extraction/sentence-transformers/'
 HUGGINGFACE_QA = 'https://api-inference.huggingface.co/pipeline/question-answering/deepset/electra-base-squad2'
-HUGGINGFACE_HEADER = {'Authorization': f'Bearer {HUGGINGFACE_API_KEY}'}
 
 RETRIEVER_MODEL_ID = 'multi-qa-mpnet-base-cos-v1'
 LITSEARCH_MODEL_ID = 'allenai-specter'
 READER_MODEL_ID = 'deepset/electra-base-squad2'
 
 
+def api_key():
+    key: str = os.getenv('HUGGINGFACE_API_KEY')
```

Comment:
this will now only blow up when someone actually searches for something, rather than when the server starts. A dubious improvement :D

Reply:
3 apps is doable. I'll have a look at it once the rest gets settled down :/